low-coverage/readme.md

```diff
@@ -0,0 +1,16 @@
+# Remove Low Coverage Fields
+
+This script removes all the fields that have coverage less than a
+given threshold coverage. For example, if given a dataset of 1000 rows
+and a threshold value of 0.10, all fields with more than 900 missing
+values will be eliminated.
+
+#Inputs
+
+-  The dataset you wish to alter
+
+- A threshold value between 0 and 1
```

Comment:
This is a good idea. I think I have it working (taking the integer to mean the maximum allowed missing and the ratio to mean the minimum percent covered), but I sometimes get turned around so another pair of eyes might be useful.